src/main/java/org/quicktheories/quicktheories/WithQuickTheories.java

```diff
@@ -0,0 +1,81 @@
+package org.quicktheories.quicktheories;
+
+import org.quicktheories.quicktheories.generators.ArbitraryDSL;
+import org.quicktheories.quicktheories.generators.ArraysDSL;
+import org.quicktheories.quicktheories.generators.BigDecimalsDSL;
+import org.quicktheories.quicktheories.generators.BigIntegersDSL;
+import org.quicktheories.quicktheories.generators.BooleansDSL;
+import org.quicktheories.quicktheories.generators.CharactersDSL;
+import org.quicktheories.quicktheories.generators.DatesDSL;
+import org.quicktheories.quicktheories.generators.DoublesDSL;
+import org.quicktheories.quicktheories.generators.FloatsDSL;
+import org.quicktheories.quicktheories.generators.IntegersDSL;
+import org.quicktheories.quicktheories.generators.ListsDSL;
+import org.quicktheories.quicktheories.generators.LocalDatesDSL;
+import org.quicktheories.quicktheories.generators.LongsDSL;
+import org.quicktheories.quicktheories.generators.SourceDSL;
+import org.quicktheories.quicktheories.generators.StringsDSL;
+
+public interface WithQuickTheories {
+  
+  default QuickTheory qt() {
+    return QuickTheory.qt();
+  }
+  
+  public default LongsDSL longs() {
```

Comment:
Inconsistent usage of `public` in interfaces (in other interfaces they are omitted, the same like in `qt()` method).